src/hardware/voodoo.cpp

```diff
@@ -6996,6 +7054,40 @@ static constexpr uint32_t voodoo_r(const uint32_t addr)
 	return 0xffffffff;
 }
 
+static int get_num_threads()
+{
+	auto get_auto_threads = []() {
+		// Sweet spot found to be seven threads based on benchmarks:
+		// https://github.com/schellingb/dosbox-pure/issues/300#issuecomment-2144050898
+		constexpr auto MinThreads = 1;
+		constexpr auto MaxThreads = 7;
+		return std::clamp(SDL_GetCPUCount() - 1, MinThreads, MaxThreads);
```

Comment:
Then here set `MinThreads` to 0 to remove the special case of `no`.  This also has the benefit of a single core CPU not spawning any additional worker threads which would probably only hurt performance.

Reply:
Nice! Added.